dsc/assertion.dsc.resource.json

```diff
@@ -7,14 +7,17 @@
     "executable": "dsc",
     "args": [
       "config",
-      "test"
+      "--as-group",
```

Comment:
It would help if there were short examples of format for both `--as-group` and `--as-get` in PR description (and docs).

Reply:
I provided descriptions of those two switches in the PR description.  They won't be doc'd since they are hidden and intended to be only used by dsc itself.